particle-iot/device-os
# Merge '[Electron] Optional feature to use HSE/LSI as RTC clock source instead of LSE (external 32KHz XTAL)' into develop

### Description

This PR contains the commits ported retroactively from `2.0.2` to `develop`. Please see https://github.com/particle-iot/device-os/pull/2354/

### Steps to Test

- Same steps to verify as https://github.com/particle-iot/device-os/pull/2354/
- Run on-device tests

### References

https://github.com/particle-iot/device-os/pull/2354/

---

### Completeness
